examples/simple_demo/Main.cc

```diff
@@ -118,6 +118,15 @@ void buildScene(ScenePtr _scene)
   box->SetMaterial(blue);
   root->AddChild(box);
 
+  // create ellipsoid visual
+  VisualPtr ellipsoidVisual = _scene->CreateVisual();
+  auto ellipsoid = _scene->CreateEllipsoid();
+  ellipsoid->SetRadii(ignition::math::Vector3d(1.2, 0.7, 0.5));
```

Comment:
This example needs to be updated now that there's no ellipsoid class. The ellipsoid's size will be defined using `SetLocalScale`, the same way as the sphere above.

Which brings the question of whether special API is needed for the ellipsoid at all. Despite its name, `ign-rendering`'s "sphere" already behaves like an "ellipsoid", because it can be scaled in 3 dimensions.

At this point, my suggestion would be to:
* remove all the ellipsoid logic added in this PR 
* update `CreateSphere`'s documentation to say that it can be used to create ellipsoids
* update this example to create an "ellipsoid" through the `CreateSphere` function

Reply:
Thank you for the bullet points :construction_worker: